public/chassis.css

```diff
@@ -0,0 +1,426 @@
+}
+
+.btn-float:focus {
+  outline: none;
+  border: 4px solid var(--color-4);
+}
+
+/* 
+ // Card
+*/
+
+.card {
+  width: 200px;
+  height: 300px;
+  padding: 0.8rem;
+  border-radius: 5px;
+  box-shadow: 0 0 10px var(--color-7);
+}
+
+.card-badge {
+  position: relative;
+  bottom: 305px;
+  left: 185px;
+  background-color: red;
+  height: 22px;
+  width: 22px;
+  border-radius: 50%;
+  text-align: center;
+  color: white;
+  font-weight: bold;
+}
+
+.title-product {
+  margin-top: 0.5rem;
+  font-size: 0.9rem;
+  font-weight: bold;
+}
+
+.price-product {
+  margin-top: 0.5rem;
+  font-weight: bold;
+}
+
+.card-text-overlay {
+  font-weight: bold;
+  width: fit-content;
+  background-color: red;
+  padding: 0.1rem 0.3rem;
+  border-radius: 5px;
+  color: white;
+  font-size: 0.9rem;
+  position: relative;
+  bottom: 280px;
+  right: 5px;
+}
+
+.card-horizontal {
+  width: 300px;
+  height: 200px;
+}
+
+.img-responsive {
+  width: 100%;
+  height: auto;
+}
+
+.img-container {
+  width: 300px;
+}
+
+.img-round {
+  border-radius: 20px;
+}
+
+.textbox {
+  width: 250px;
+  padding: 0.5rem;
+  resize: none;
+  border-radius: 5px;
+  border: 4px solid var(--color-2);
+}
+
+.textbox:focus {
+  outline: none;
+}
+
+.list-spaced {
+  list-style: none;
+  display: flex;
+  justify-content: space-evenly;
+  padding-inline-start: 0;
+}
+
+.list-spaced > li {
+  margin: 0 1rem;
+}
+
+.list-stacked {
+  list-style: none;
+  padding-inline-start: 0;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1rem;
+}
+
+.list-stacked > li {
+  margin-bottom: 0.5rem;
+}
+
+.nav-desktop {
+  width: 100%;
+}
+
+.nav-list {
+  justify-content: space-around;
+}
+
+.nav-item {
+  border-right: 1px solid black;
+  width: 100%;
+  text-align: center;
+}
+
+.nav-item:last-child {
+  border: none;
+}
+
+.modal-bg {
+  width: 100vw;
+  height: 100vh;
+  background-color: #ffffffb4;
+  position: fixed;
+  top: 0;
+  left: 0;
+  justify-content: center;
+  align-items: center;
+  display: none;
+  z-index: 2;
+}
+
+.modal {
+  background-color: white;
+  width: 50vw;
+  height: 60vh;
+  border: 1px solid var(--color-2);
+  border-radius: 5px;
+  box-shadow: 0 0 10px #d1d5db;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.grid-two {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.grid-three {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-column-gap: 1rem;
+}
+
+.heading-1 {
+  font-weight: bold;
+  font-size: 1.5rem;
+  margin: 0.5rem 0;
+}
+
+.heading-2 {
+  font-weight: bold;
+  font-size: 1.3rem;
+  margin: 0.3rem 0;
+}
+
+.heading-3 {
+  font-weight: bold;
+  font-size: 1rem;
+  margin: 1rem 0;
+  margin-bottom: 0.5rem;
+}
+
+.txt-small {
+  font-size: 0.7rem;
+}
+
+.txt-grey {
+  color: grey;
+}
+
+.txt-center {
+  text-align: center;
+}
+
+.toast {
+  border: 1px solid #00ffa2;
+  padding: 0.5rem;
+  border-radius: 5px;
+  width: fit-content;
+  position: fixed;
+  left: 1rem;

```

Comment:
The Toast message overlaps on the Navigation menu and doesn't look good . You can fix that by **left: 40rem;** or adjust accordingly.